components/layouts/sidebar.tsx

```diff
@@ -1,5 +1,77 @@
+'use client';
+
+import { BookIcon, HomeIcon, SettingIcon } from '@/assets/icons';
+import { Logo } from '@/assets/svg-images';
+import Image from 'next/image';
+import { useRouter } from 'next/navigation';
+import { useState } from 'react';
+
+const sidebarItem = [
```

Comment:
It is better to move this constant into `constants` folder, i.e. utils -> constants.
Also we can rename it as `sidebarItems`